### PR TITLE
Re-do PR #14685 for alt-text inventories.

### PR DESCRIPTION
**Pull request by tvo318**
##### SUMMARY
This PR addresses all the review changes from PR #14685 for the Inventories chapter of the User Guide.
No review needed, since I reviewed and suggested these changes from the parent PR; however, approval needed to merge.

##### ISSUE TYPE
 - Bug, Docs Fix or other nominal change

##### COMPONENT NAME
 - Docs

##### AWX VERSION
latest

### CODE REVIEW

**Review by samccann — APPROVED** [review]

File: docs/docsite/rst/userguide/inventories.rst [hunk 1]
```diff
@@ -186,24 +189,29 @@ If you wanted to add an ansible fact of
 In the search field, enter ``ansible_processor[]="GenuineIntel"`` (no extra spaces or ``__`` before the value) and press **[Enter]**.
 
 .. image:: ../common/images/inventories-smart-define-host-filter-facts.png
+   :alt: Example of an advanced search for host filter ansible facts
```
Comment (oraNod) [inline docs/docsite/rst/userguide/inventories.rst]:
```suggestion
   :alt: Example of an advanced search for host filter Ansible facts
```

File: docs/docsite/rst/userguide/inventories.rst [hunk 2]
```diff
@@ -186,24 +189,29 @@ If you wanted to add an ansible fact of
...
 
 The resulting search criteria for the specified ansible fact populates in the lower part of the window.
 
 .. image:: ../common/images/inventories-smart-define-host-filter-facts2.png
+   :alt: Selected search criteria for host filter ansible facts listed below search field
```
Comment (oraNod) [inline docs/docsite/rst/userguide/inventories.rst]:
```suggestion
   :alt: Selected search criteria for host filter Ansible facts listed below search field
```

File: docs/docsite/rst/userguide/inventories.rst [hunk 3]
```diff
@@ -186,24 +189,29 @@ If you wanted to add an ansible fact of
...
 
 3. Click **Select** to add it to the **Smart host filter** field.
 
 .. image:: ../common/images/inventories-smart-create-filter-added.png
+   :alt: Specified search criteria for host filter ansible facts shown in the Smart host filter field of the Create new smart inventory window
```
Comment (oraNod) [inline docs/docsite/rst/userguide/inventories.rst]:
```suggestion
   :alt: Specified search criteria for host filter Ansible facts shown in the Smart host filter field of the Create new smart inventory window
```

File: docs/docsite/rst/userguide/inventories.rst [hunk 4]
```diff
@@ -186,24 +189,29 @@ If you wanted to add an ansible fact of
...
 
 4. Click **Save** to save the new Smart Inventory.
 
 The Details tab of the new Smart Inventory opens and displays the specified ansible facts in the **Smart host filter** field.
 
 .. image:: ../common/images/inventories-smart-create-details.png
+   :alt: Details tab of the new Smart Inventory displaying the specified ansible facts in the Smart host filter field
```
Comment (oraNod) [inline docs/docsite/rst/userguide/inventories.rst]:
```suggestion
   :alt: Details tab of the new Smart Inventory displaying the specified Ansible facts in the Smart host filter field
```

File: docs/docsite/rst/userguide/inventories.rst [hunk 5]
```diff
@@ -186,24 +189,29 @@ If you wanted to add an ansible fact of
...
 
 5. From the Details view, you can edit the **Smart host filter** field by clicking **Edit** and delete existing filter(s), clear all existing filters, or add new ones.
 
 .. image:: ../common/images/inventories-smart-define-host-filter-facts-group.png
+   :alt: Specified search criteria consisting of host filter ansible facts and groups listed below search field
```
Comment (oraNod) [inline docs/docsite/rst/userguide/inventories.rst]:
```suggestion
   :alt: Specified search criteria consisting of host filter Ansible facts and groups listed below search field
```

File: docs/docsite/rst/userguide/inventories.rst [hunk 6]
```diff
@@ -1152,6 +1195,7 @@ This inventory source uses a cluster that is able to deploy OpenShift (OCP) virt
 
 
   .. image:: ../common/images/inventories-create-source-ocpvirt-example.png
+   :alt: Inventories create source Openshift virtualization example
```
Comment (oraNod) [inline docs/docsite/rst/userguide/inventories.rst]:
```suggestion
   :alt: Inventories create source OpenShift virtualization example
```

File: docs/docsite/rst/userguide/inventories.rst [hunk 7]
```diff
@@ -1290,10 +1334,13 @@ The Run command window opens.
 |ad hoc-commands-inventory-run-command|
 
 .. |ad hoc-commands-inventory-run-command| image:: ../common/images/ad-hoc-commands-inventory-run-command.png
+   :alt: Ad hoc commands inventory run command
+
 
 4. Click **Next** to choose the |ee| you want the ad-hoc command to be run against.
 
 .. image:: ../common/images/ad-hoc-commands-inventory-run-command-ee.png
+   :alt: Ad Hoc Run command dialog prompting for Execution Environments with two listed to choose from
```
Comment (oraNod) [inline docs/docsite/rst/userguide/inventories.rst]:
```suggestion
   :alt: Ad hoc run command dialog prompting for Execution Environments with two listed to choose from
```

**Review by oraNod — APPROVED** [review]
Thanks @tvo318

**Pull request merged**